reference/ps-modules/PSScriptAnalyzer/Invoke-ScriptAnalyzer.md

```diff
@@ -514,7 +514,7 @@ following keys:
 
 The keys and values in the profile are interpreted as if they were standard parameters and values of
 `Invoke-ScriptAnalyzer`, similar to splatting. For more information, see
-[about_Splatting](/powershell/module/microsoft.powershell.com/about/about_splatting).
+[about_Splatting](/powershell/module/microsoft.powershell.core/about/about_splatting).
```

Comment:
PSSA docs use `core` in the URL, which one should we use? https://github.com/PowerShell/PSScriptAnalyzer/blob/6790ebba18d6732fd6edf008c48e566adb508641/docs/Cmdlets/Invoke-ScriptAnalyzer.md?plain=1#L517
It seems this was maybe changed to `com` when PSSA docs were migrated to MicrosoftDocs but not synced back to PSSA. Should I revert this change and sync the `com` change back to PSSA?

Reply:
@bergmeister it looks like the switch to `com` was a mistake because that link is definitely broken - it should be `microsoft.powershell.core` which resolves correctly.

Reply:
Yes, you're right, thanks.